src/language-css/printer-postcss.js

```diff
@@ -534,7 +535,7 @@ function genericPrint(path, options, print) {
 
     case "value-colon": {
       const { previous } = path;
-      return [
+      return label("colon", [
```

Comment:
This used somewhere?

Reply:
https://github.com/prettier/prettier/pull/16570/commits/24ddbd492de16ce77cd784c08e17e43f2ada532d
Grouping it changes some outputs. It looks slightly better to me. Would you take a look?